src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -173,11 +242,6 @@ public void parse_invalidValue_failure() {
                 + TUTORIAL_GROUP_DESC_BOB + SOCIAL_HANDLE_DESC_BOB + REMARK_DESC_BOB
                 + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Email.MESSAGE_CONSTRAINTS);
 
-        // invalid nationality
-        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + INVALID_NATIONALITY_DESC
```

Comment:
I was wondering if this should be kept as there is a prefix for nationality but the argument is left blank (i.e. just `nat/` instead of `nat\Singaporean`)? I would would imagine that it would be invalid as well.